coordinator/client/connection.go

```diff
@@ -39,12 +39,20 @@ type Node interface {
 	SetVersion(interface{})
 }
 
+type Transaction interface {
+	Create(path string, node Node)
```

Comment:
If they ever do anything more complicated than they are, yes maybe, but they can't if we're going to try to implement chaining.